packages/sitecore-jss-vue/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["plugin:vue/recommended", "../../.eslintrc"],
+  "rules": {
+    // TODO: fix errors
```

Comment:
Same here - wouldn't consider this a TODO

Reply:
Removed